Formula/i/icecast.rb

```diff
@@ -9,11 +9,9 @@ class Icecast < Formula
 
   livecheck do
     url "https://ftp.osuosl.org/pub/xiph/releases/icecast/?C=M&O=D"
-    regex(%r{href=(?:["']?|.*?/)icecast[._-]v?(\d+(?:\.\d+)+)\.t}i)
+    regex(%r{href=(?:["']?|.*?/)icecast[._-]v?((?!.*999)\d+(?:\.\d+)+)\.t}i)
```

Comment:
```suggestion
    regex(%r{href=(?:["']?|.*?/)icecast[._-]v?(\d+\.\d+(?:\.(?:\d|[1-8]\d+)(?:\.\d+)*)?)\.t}i)
```

I've seen unstable versions with a 999 patch in projects that use a 90+ patch to indicate unstable/development versions (it seems like a somewhat sloppy way of saying "this is _defffinitely_ a development version, trust me"), so there may be something to be said for using the 90+ pattern here. This would avoid 999 but in a less explicit way that would also cover 90+ patch versions.